activesupport/lib/active_support/event_reporter.rb

```diff
@@ -257,6 +257,7 @@ def initialize(*subscribers, raise_on_error: false, tags: nil)
     #   name: String (The name of the event)
     #   payload: Hash, Object (The payload of the event, or the event object itself)
     #   tags: Hash (The tags of the event)
+    #   context: Hash (The context of the event)
```

Comment:
Should this be added to `notify` examples too?

Reply:
Okay, went back other the other examples and filled out any missing attributes. Thanks for the review @skipkayhil ! ❤️